docs/server.md

```diff
@@ -52,9 +52,10 @@ optional arguments:
   -h, --help            show this help message and exit
   --models KEY1=VAL1 KEY2=VAL2... [KEY1=VAL1 KEY2=VAL2... ...]
                         Models to be deployed using name=model_location
-                        format. Location can be a URL or a local path to a
-                        .model file. Name is arbitrary and used as the API
-                        endpoints base name.
+                        format. Location can be a URL, a local path to a
```

Comment:
Why are we changing this? I think we should drop this change.

Reply:
We can serve from folder now.

Reply:
Got it. Cool.